components/CreateTodo.js

```diff
@@ -0,0 +1,111 @@
+
+import toast from "./Toast.js";
+import timeCalculation from "../utilities/TimeCalculation.js";
+
+const createTodo = () => {
+  // const buttonCreate = document.querySelector('#buttonCreate');
+  // const createTodo = document.querySelector('#createTodo');
+  // const cancelTodoButton = document.querySelector('#cancelTodo');
+  // const textArea = document.querySelector('#enterTodo');
+
+  // const addTodoBtn = document.querySelector('#addTodo');
+  // const todoWrapper = document.querySelector('#todoWrapper');
+
+  // const openCreateTodo = () => {
+  //   createTodo.classList.add('open-create-todo');
+  //   textArea.focus();
+  // }
+
+  // const cancelTodo = () => {
+  //   createTodo.classList.remove('open-create-todo');
+  //   textArea.value = '';
+  // }
+
+  // buttonCreate.addEventListener('click', openCreateTodo);
+  // cancelTodoButton.addEventListener('click', cancelTodo);
+
+  // const emptyNotice = document.createElement('h2');
+
+  // if(todoWrapper.innerHTML === '') {
+  //   emptyNotice.innerText = 'Todo list is empty';
+  //   todoWrapper.appendChild(emptyNotice)
+  // }
+
+  let todos = [];
```

Comment:
We should not declare the source of data here. `todos` array should be initialised higher up the tree / as close to root as possible. 

So can we move it to `entry.js` and use it there?

```js
// entry.js file
import blah blah

// We declare the source of data
let todos = []

// We add the eventListeners
const addButton.addEventListener("click", handleAddTodo)